### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by stefanpenner**

### CODE REVIEW

File: blueprints/app/files/bower.json [hunk 1]
```diff
@@ -3,9 +3,9 @@
   "dependencies": {
     "ember": "1.11.1",
     "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
-    "ember-cli-test-loader": "ember-cli-test-loader#0.1.4",
+    "ember-cli-test-loader": "ember-cli-test-loader#0.1.3",
```
Comment (rwjblue) [inline blueprints/app/files/bower.json]: Why are we downgrading? Is there a regression?

Comment (stefanpenner) [inline blueprints/app/files/bower.json]: i accidentally upgraded this library, whne i meant to upgrade another. 

0.1.3 is the latest version: https://github.com/ember-cli/ember-cli-test-loader/blob/master/bower.json#L4

Comment (rwjblue) [inline blueprints/app/files/bower.json]: Ahh, thanks for the explanation,  sorry for the noise

Comment (stefanpenner) [inline blueprints/app/files/bower.json]: nope, thanks for the second set of eyes.

**Pull request merged**